Sources/DoubleExtensions.swift

```diff
@@ -14,6 +14,9 @@ extension Double {
 
     /// EZSE: Converts Double to Int
     public var toInt: Int { return Int(self) }
+    
+    /// EZSE: Converts Double to CGFloat
+    public var toCGFloat: CGFloat { return CGFloat(self) }
```

Comment:
Is there a possibility of precision loss in this conversion? (E.g. Very large or high number of fraction digits causing some of those to be lost). If yes, we need to write some tests around that to document the edge case.